distributed/tests/test_steal.py

```diff
@@ -1621,19 +1630,30 @@ async def _run(
                 **kwargs,
             )
 
+        default_task_durations = {
+            compose_task_prefix(deps): "1s"
+            for tasks in task_placement
+            for deps in tasks
+        }
+
         gen_cluster(
             client=True,
             nthreads=[("", 1)] * len(task_placement),
             config=merge(
                 NO_AMM,
                 config or {},
                 {
-                    "distributed.scheduler.unknown-task-duration": "1s",
+                    "distributed.scheduler.default-task-durations": default_task_durations,
```

Comment:
Why is this change needed?

Reply:
This was changed to avoid the reliance on stealing tasks of unknown duration. Instead, we now provide default durations for all relevant tasks. This concern was initially voiced here:

> The test implicitly relies on tasks of unknown duration to be stolen (https://github.com/dask/distributed/pull/5572). It should be changed not to rely on this specific use case.

 
https://github.com/dask/distributed/pull/7243#discussion_r1012837335